src/csharp/Microsoft.Spark.E2ETest/BridgeTests/JvmBridgeTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Spark.Sql;
+using Xunit;
+
+namespace Microsoft.Spark.E2ETest.BridgeTests
+{
+    [Collection("Spark E2E Tests")]
+    public class JvmBridgeTests
+    {
+        [Fact]
+        public void TestSparkExceptionReturned()
+        {
+            using var session = SparkSession.GetDefaultSession();
```

Comment:
Can you use the `SparkSession` from `SparkFixture`? You can check other E2E tests such as https://github.com/dotnet/spark/blob/master/src/csharp/Microsoft.Spark.E2ETest/IpcTests/BroadcastTests.cs#L30

Reply:
Done